pkg/operator/register.go

```diff
@@ -190,9 +192,20 @@ func validateAndMigrateConfigFile(path string) (*types.OperatorConfigNew, error)
 			return nil, err
 		}
 	}
+	operatorCfg.ELAVSDirectoryAddress = getAVSDirectoryAddress(operatorCfg.ChainId)
 	return &operatorCfg, nil
 }
 
+func getAVSDirectoryAddress(chainID big.Int) string {
+	chainIDInt := chainID.Int64()
+	chainMetadata, ok := utils.ChainMetadataMap[chainIDInt]
+	if !ok {
+		return ""
```

Comment:
neat: shall this be nil or function signature returns err?

Reply:
yea I might as well return error if it's some random chainID. Will update. Good catch